src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task{
+    protected String dueDate;
+
+    public Deadline(String task, String dueDate) {
```

Comment:
Maybe you can include Javadoc comments for each class method? Just a short description of param and return values will make it better.